docs/hosts/cisco.md

```diff
@@ -7,7 +7,7 @@ can get more details on this from their
 Wind River Linux support came to Beaker through testing Cisco network
```

Comment:
This sentence gives me pause, TBH - a lot of the logic below is specific to Cisco's OS and is probably not applicable to WRL in general. Does Beaker really support arbitrary WRL distros?